spaceship/lib/spaceship/connect_api/tunes/tunes.rb

```diff
@@ -623,9 +623,9 @@ def patch_app_store_review_detail(app_store_review_detail_id: nil, attributes: {
       # appStoreVersionLocalizations
       #
 
-      def get_app_store_version_localizations(filter: {}, includes: nil, limit: nil, sort: nil)
-        params = Client.instance.build_params(filter: filter, includes: includes, limit: limit, sort: sort)
-        Client.instance.get("appStoreVersionLocalizations", params)
+      def get_app_store_version_localizations(app_store_version_id: nil, limit: nil)
+        params = Client.instance.build_params(filter: nil, includes: nil, limit: limit, sort: nil)
```

Comment:
Why are `filter` and `includes` still required here?

Reply:
Technically they're not, `build_params` has a default value of `nil` for them. We should totally be able to simplify this.

Reply:
Basically, I did it this way to keep the same style that has been used in other parts of the codebase, where they do things like this for instance:
```
def get_app(app_id: nil, includes: nil)
  params = Client.instance.build_params(filter: nil, includes: includes, limit: nil, sort: nil)
  Client.instance.get("apps/#{app_id}", params)
end
```
even though they should be able to just specify `includes`. It should make this PR more "acceptable" to fastlane maintainers should we decide to push it to upstream.